src/main/java/lotto/utility/RandomNumberGenerator.java

```diff
@@ -0,0 +1,15 @@
+package lotto.utility;
+
+import java.util.Random;
+
+public class RandomNumberGenerator {
+
+    private static final Random random = new Random();
```

Comment:
`static final`를 통해 상수로 선언하셨으니, 관례에 따라 이름을 대문자로 작성해보면 어떨까요?
```suggestion
    private static final Random RANDOM = new Random();
```